### ISSUE TITLE
First bullet point in LICENSE should not be a bullet

### ISSUE THREAD

**Issue by andreww** [andreww/fox#45]
It looks like the first bullet point in the LICENSE file should not be a bullet point. Is this why github does not pick up the license?

**Comment by zbeekman** [comment]
Yes, in general there's a lot of excess text in the license file. You can get the correct template for the BSD 3 clause license at: 

https://choosealicense.com/licenses/bsd-3-clause/

It would be great if you could clean this up so it's automatically detected. It took more sleuthing than I would have preferred to figure out that it's BSD 3. I can create a PR if you like.
